packages/babel-plugin/src/properties/chunkName.js

```diff
@@ -132,7 +132,6 @@ export default function chunkNameProperty({ types: t }) {
     const values = getExistingChunkNameComment(callPath)
 
     if (!agressiveImport && values) {
-      addOrReplaceChunkNameComment(callPath, values)
```

Comment:
this change should be separated from `crossEnv`, and have a clear test to prove it's necessity  and correctness.

Reply:
This seems to be github automatic behavior. I have run the tests locally